src/wasm-linker.cpp

```diff
@@ -210,6 +210,15 @@ void Linker::layout() {
     auto* func = out.wasm.getFunction(name);
     func->type = ensureFunctionType(getSig(func), &out.wasm)->name;
   }
+
+  // Export malloc and free whenever availble. JavsScript version of malloc has
```

Comment:
Oh, I was mistaken about the prefix _ thing. emscripten creates
var _FUNCNAME = Module[_FUNCNAME] = asm[FUNCNAME]
for every function that is exported from a wast file anyway, so we don't need to modify something frmo emscripten side for this. What I did is we need to export malloc and free from wast side, and I don't think this is related with _ prefix thing. Is it?